hdf5-vol-daos.spec

```diff
@@ -197,6 +197,10 @@ done
 %endif
 
 %changelog
+* Wed Jan 27 2021 Maureen Jean <maureen.jean@intel.com> - 1.1.0rc1.g9eebd90
+- Update hdf5-vol 1.1.0rc1. use commit vol-daos g9eebd90 
```

Comment:
Don't need to mention git hashes when using a tag/release.  It only adds confusion.